inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/strided_slice.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace CPUTestUtils;
+using namespace InferenceEngine;
+
+namespace SubgraphTestsDefinitions {
+
+class StridedSliceSubgraphTest : public CPUTestsBase, virtual public LayerTestsUtils::LayerTestsCommon {
+public:
+    void SetUp() override {
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        inPrc = outPrc = Precision::FP32;
+        const std::vector<size_t> inputShape = {1, 32, 112, 112};
+        const auto type = ngraph::element::f32;
+
+        auto inputParams = ngraph::builder::makeParams(type, {inputShape, inputShape});
+
+        const std::vector<int64_t> begin = {0, -28, 0, 0};
```

Comment:
Could you please add more tests covered all cases when we cannot create blocked layout?

For example,
1. there is `shrink_axis_mask` or `new_axis_mask` (not equal input and output dims)
2. srcDims[1] % {8, 16} != 0
3. stride[1] != 1
etc.

Moreover I think we should create these tests in CPU single layer tests (update existing tests) but not in subgraph because I don't see that these tests are somewhat different to create it in subgraph tests. I see only one node StridedSlice, not subgraph. What do you think?